src/transformers/data/datasets/language_modeling.py

```diff
@@ -99,3 +101,119 @@ def __len__(self):
 
     def __getitem__(self, i) -> torch.Tensor:
         return torch.tensor(self.examples[i], dtype=torch.long)
+
+class AlbertTextDataset(Dataset):
```

Comment:
That dataset is essentially the `LineByLineTextDataset` with the SOP task, right? In that case, could it be renamed to `LineByLineWithSOPTextDataset` or something along those lines? Again, less constrained to a single model.

Reply:
Same as before, if the name changes, don't forget to adapt the docstrings below :)

Reply:
updated both name and docstrings.